.github/workflows/julia.yml

```diff
@@ -0,0 +1,57 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: Julia
+on:
+  push:
+    paths:
+      - '.github/workflows/julia.yml'
+      - 'ci/docker/*_julia.dockerfile'
+      - 'julia/**'
+  pull_request:
+    paths:
+      - '.github/workflows/julia.yml'
+      - 'ci/docker/*_julia.dockerfile'
+      - 'ci/docker/**'
+      - 'julia/**'
+
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
```

Comment:
We use `${CPU} ${PLATFORM} ${LANGUAGE} ${OPTIONAL_NOTE}` format in our jobs.
For example:

  * `AMD64 Ubuntu 18.04 C++ ASAN UBSAN` where `${CPU}=AMD64`, `${PLATFORM}=Ubuntu 18.04`, `${LANGUAGE}=C++` and `${OPTIONAL_NOTE}=ASAN UBSAN`
  * `AMD64 Conda Python 3.8 Without Pandas` where `${CPU}=AMD64`, `${PLATFORM=Conda`, `${LANGUAGE}=Python 3.8` and `${OPTIONAL_NOTE}=WIthout Pandas`

Can we follow the style?

Reply:
Updated